matrix.json

```diff
@@ -31,19 +31,35 @@
                     },
                     {
                         "type": "crypto",
-                        "ticker": "BCH"
+                        "ticker": "BTCV"
```

Comment:
I will merge this but note that new currencies that don't exist by default in the list of 186 in the existing currencies table (BTCV and countless others) will not automatically make it into user-searches. 

There is a small amount of development work I have been holding off on, to make this happen - namely: A centralised resource of all coins to parse. Seeing as not all exchanges list all coins, this is going to be a little hard to come by I've seen this which can be fetched an cached daily (>8Mb!). Do you know of any other resources? Alts are not my speciality.

Reply:
I hate alts and prefer to call them shitcoins. I wouldn't lock this field to any list and certainly not an 8MB list. Just let the wallets claim whatever symbols and users search for whatever.

Also if this matrix file is supposed to catch on, give the wallet providers more liberties than what your site can handle. I would probably list some shit-coin-count on WalletScrutiny but certainly not the logos of all those coins or full names etc.